### PR TITLE
Remove ConfigureDefaultOptions

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

**Comment by Tratcher** [comment]
@DamianEdwards @danroth27 are we sure about this one? The default binders do not work for Kestrel.

**Comment by danroth27** [comment]
Yes, the config support for kestrel is getting pulled for 2.0.

File: src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj [hunk 1]
```diff
@@ -14,7 +14,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Certificates.Configuration.Sources" Version="$(AspNetCoreVersion)" PrivateAssets="All" />
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj]: No longer used?
@javiercn are you going to use Microsoft.AspNetCore.Certificates.Configuration.Sources in Identity? If not, should it be removed from Hosting for now?

Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj]: Remove Kestrel.Https as well, it was only added for config binding

**Review by Tratcher — CHANGES_REQUESTED** [review]

**Comment by HaoK** [comment]
Reverted the original change instead https://github.com/aspnet/KestrelHttpServer/issues/1875

**Pull request closed without merging**